src/main/java/io/debezium/connector/yugabytedb/HelperBeforeImageModes.java

```diff
@@ -21,9 +21,19 @@ public enum BeforeImageMode implements EnumeratedValue {
         FULL_ROW_NEW_IMAGE("FULL_ROW_NEW_IMAGE"),
 
         /**
-         * MODIFIED_COLUMNS_OLD_AND_NEW_IMAGES mode, old and new images of modified column
+         * CHANGE_OLD_NEW mode, old and new images of modified column
```

Comment:
Should we keep the `ALL` and `MODIFIED_COLUMNS_OLD_AND_NEW_IMAGES` to maintain backward compatibility?

Reply:
Yes, we should. I commented the same on the server side diff. It is required since we store this in sys-catalog on disk.

Reply:
Restored. Have added [Deprecated] in comment